ginipaybank/src/main/java/net/gini/pay/bank/capture/digitalinvoice/DigitalInvoiceFragment.kt

```diff
@@ -45,7 +45,8 @@ private const val TAG_WHAT_IS_THIS_DIALOG = "TAG_WHAT_IS_THIS_DIALOG"
  * - the line items are updated according to the user's modifications.
  *
  * You should show the `DigitalInvoiceFragment` when the
- * [AnalysisFragmentListener.onProceedToReturnAssistant()] is called.
+ * [AnalysisFragmentListener.onExtractionsAvailable()] is called and you have validated the compound extractions using the
```

Comment:
`and you have validated` to `when you have validated`?

Reply:
What do you think about this?
```
Before showing the `DigitalInvoiceFragment` you should validate the compound extractions 
using the [LineItemsValidator]. These extractions are 
returned in the [AnalysisFragmentListener.onExtractionsAvailable()] 
listener method.
```